BRPeer.h

```diff
@@ -52,17 +52,11 @@
 extern "C" {
 #endif
 
-#if LITECOIN_TESTNET
-#define STANDARD_PORT 19335
-#else
-#define STANDARD_PORT 9333
-#endif
-
 #define SERVICES_NODE_NETWORK 0x01 // services value indicating a node carries full blocks, not just headers
 #define SERVICES_NODE_BLOOM   0x04 // BIP111: https://github.com/bitcoin/bips/blob/master/bip-0111.mediawiki
 #define SERVICES_NODE_BCASH   0x20 // https://github.com/Bitcoin-UAHF/spec/blob/master/uahf-technical-spec.md
     
-#define BR_VERSION "175"
+#define BR_VERSION "2.1"
```

Comment:
The BR_VERSION was intended to be the same as the release build version.

Reply:
Oh I was wondering the same. 
So whenever we release a new Android build version, we must update the BR_VERSION too ?
If so having the same branch for Android/iOS will be an issue since they will not have the same release version. Thoughts? @losh11

Reply:
If you're using the same branch, then I'd suggest tagging loafwallet-core releases with it's own version number, and using that here. It's just useful for others to see what version their peer is connecting to.

Reply:
yeah def make sense. `BR_VERSION` has to be the version of  `loafwallet-core release build`

Reply:
Since now both Android/iOS app are using the same loafwallet-core codebase, we will keep BR_VERSION = 2.1 which the current version used on Litewallet iOS. thoughts?